hazelcast/src/test/java/com/hazelcast/collection/impl/queue/QueueStoreTest.java

```diff
@@ -79,6 +80,28 @@ public void testQueueStoreLoadMoreThanMaxSize() {
     }
 
     @Test
+    public void testRemoveAll() {
+        int maxSize = 2000;
+        final Config config = new Config();
+        final QueueStoreConfig queueStoreConfig = new QueueStoreConfig()
+                .setStoreImplementation(new TestQueueStore())
+                .setProperty("bulk-load", String.valueOf(200));
+        config.getQueueConfig("testQueueStore")
+                .setMaxSize(maxSize)
+                .setQueueStoreConfig(queueStoreConfig);
+        final HazelcastInstance instance = createHazelcastInstance(config);
+        final IQueue<Object> queue = instance.getQueue("testQueueStore");
+
+        final ArrayList<Integer> data = new ArrayList<Integer>();
+        for (int i = 0; i < maxSize; i++) {
+            queue.add(i);
+            data.add(i);
+        }
+        queue.removeAll(data);
+        assertEquals(0, queue.size());
```

Comment:
Something like this:

```
boolean result =  queue.removeAll(data);

assertTrue(result);
assertEquals(0, queue.size());
assertEquals(asList(1,2,3,4), data)
```
So that the content of 'data' is verified and also the return value.